### DISCUSSION TITLE
Error when calling in a subdomain through an iframe.

### DISCUSSION THREAD

**Discussion by BraydenPB** [biati-digital/glightbox#429] [Q&A]
I am trying to show an iframe of my subdomains through Glightbox, I had this working perfectly fine until one day I started to receive this error: "Cross-Origin Read Blocking (CORB) blocked cross-origin response with MIME type text/html. See https://www.chromestatus.com/feature/5629709824032768 for more details."

Since I am trying to call in subdomains, I am unable to create a CORB policy for the subdomain. Does anybody have a solution for this?

You can see here in this codepen: https://codepen.io/braydenpb72/pen/OJabOOv

**Comment by gingerchew** [comment]
Looking at the documentation for the CORB, I don't think there is a way around it. I would think that you could set up something on whatever is serving from the subdomain to say "no, this domain is cool and is *supposed* to be requesting this", but I can't find anything explicitly saying there is
